jsonnet/dashboards.libsonnet

```diff
@@ -0,0 +1,276 @@
+local overrides = import 'overrideDashboard.libsonnet';
+
+local defaults = {
+    local defaults = self,
+    namespace: 'perses-dev',
+    commonLabels:: {
+        "app.kubernetes.io/component": "dashboard",
+        "app.kubernetes.io/name": "perses-dashboard",
+        "app.kubernetes.io/part-of": "perses-operator"
+    },
+    datasource: 'prometheus-datasource',
+    components: ['kubernetes', 'thanos', 'etcd', 'blackbox-exporter', 'node-exporter', 'alertmanager', 'prometheus', 'perses'],
+};
+
+function (params) {
+    local cd = self,
+    config:: defaults + params,
+
+    local alertmanagerOverview = if std.member(cd.config.components, 'alertmanager') then overrides.overrideDashboard(
+        import '../examples/dashboards/json/operator/alertmanager/alertmanager-overview.json',
```

Comment:
hmm don't we need to have all JSON dashboards in the jsonnet/ directory (e.g. jsonnet/generated) to import downstream?

Reply:
I needed to edit makefile, done now